vagrant/scripts/install_iml_local.sh

```diff
@@ -4,8 +4,12 @@ set -ex
 
 yum copr enable -y managerforlustre/manager-for-lustre-devel
 # Get latest rpmdevtools
-yum install -y https://copr-be.cloud.fedoraproject.org/results/managerforlustre/buildtools/epel-8-x86_64/01152137-rpmdevtools/rpmdevtools-8.10-7.el8.noarch.rpm
-yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+if ! yum list installed rpmdevtools-8.10-7.el8.noarch; then
```

Comment:
I don't think the check should be version locked, should it?

Reply:
I think a grep could be used here to avoid the version right?

Reply:
We want the specific version, that's the reason for that entire manipulation. It never changes.

Reply:
I'm still a bit confused. This is installing the rpm but then on line 13 it's installing rpmdevtools again.

Reply:
you can just do `yum list installed rpmdevtools`
There's no `yum remove` so having a different version isn't checked for.

Reply:
> I'm still a bit confused. This is installing the rpm but then on line 13 it's installing rpmdevtools again.

First RPM is just RPM with repository data. It's symmetric to what happens with PostgreSQL installation - first repo RPM, then RPM of package itself.

Reply:
Ah that's why it's confusing me. Any reason we can't use yum-config-manager to add the repo? http://man7.org/linux/man-pages/man1/yum-config-manager.1.html

Reply:
To be clear, i'm not trying to hold this up, when I first looked at it it looked like it was installing the same rpm twice (since I didn't know the first rpm was to install the repository). I think something like yum-config-manager would make it a little clearer if it is possible.

Reply:
@johnsonw 

`yum-config-manager` is nice, I tried what you suggested and it even checks for duplicate repositories. Problem is, it errors when there's a duplicate, and `rpm -q` doesn't show the repo RPM as installed, so the only way around that would be to silence errors from `yum-config-manager`, which isn't great as it can fail for another reason rather than duplicate repo.
```
    adm: adding repo from: https://copr-be.cloud.fedoraproject.org/results/managerforlustre/buildtools/epel-8-x86_64/01152137-rpmdevtools/rpmdevtools-8.10-7.el8.noarch.rpm
    adm: Cannot add repo from https://copr-be.cloud.fedoraproject.org/results/managerforlustre/buildtools/epel-8-x86_64/01152137-rpmdevtools/rpmdevtools-8.10-7.el8.noarch.rpm as is a duplicate of an existing repo
```